examples/by_feature/memory.py

```diff
@@ -19,7 +19,7 @@
 from accelerate import Accelerator, DistributedType
 
 # New Code #
-from accelerate.memory_utils import find_executable_batch_size
+from accelerate.utils import find_executable_batch_size
```

Comment:
This one is rather important -> let's put it in the main init as well so that users import it directly from `accelerate` (and this way they'll be immune to the next cleanup)